autoreject/tests/test_ransac.py

```diff
@@ -32,20 +32,29 @@ def test_ransac():
     # normal case
     picks = mne.pick_types(epochs.info, meg='mag', eeg=False, stim=False,
                            eog=False, exclude=[])
-
     ransac = Ransac(picks=picks, random_state=np.random.RandomState(42))
     epochs_clean = ransac.fit_transform(epochs)
     assert len(epochs_clean) == len(epochs)
+
+    # Pass string instead of array of idx
+    picks = 'eeg'
+    ransac = Ransac(picks=picks, random_state=np.random.RandomState(42))
+    epochs_clean = ransac.fit_transform(epochs[:2])
```

Comment:
you can even just do:

```suggestion
    epochs = ransac.fit(epochs[:2])
```

to make it even faster. We really just want to test the `picks`. L43 is probably tested elsewhere

Reply:
Sure. The test right above is very slow, maybe you could push a fix where only 2 epochs are used instead of the 4?